sdk/storage/azqueue/go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/Azure/azure-sdk-for-go/sdk/storage/azqueue
```

Comment:
Please be sure to run `go mod tidy` and include the generated `go.sum` file.

Reply:
done and sum file is already in there

Reply:
Oh yes some files have been previously committed, my mistake.